back-office/shop-parameters/customer-settings/titles.md

```diff
@@ -0,0 +1,69 @@
+
+A drop-down next to the field is displayed when there are several languages installed (disabled languages are displayed in the drop-down). It allows choosing in which language the field is displayed and customizing the title according to the customer's language.
+
+**Gender.** As titles may apply to both male and female customers, or neither, three radio buttons allow merchants to **specify to which category the title applies**. By default, the 'Male' gender is selected.
+
+- Male<br>
+- Female<br>
+- Neutral<br>
+
+**Image.** Users can **illustrate social titles with a picture**. Clicking 'Add file' should allow merchants to browse their files and pick the right image. Classic gender symbols are the Mars and Venus ones (♂ and ♀) and apply to the 'Mr.' and 'Mrs.' default titles.
+
+Allowed formats are: .gif, .jpg, .png., .svg, and .webp. If a different file format is uploaded and attempted to be saved, an error notification is displayed: `Image format not recognized, allowed formats are: .gif, .jpg, .png., .svg, .webp.` in `Admin.Notifications.Error`.
+
+**Image width.** Merchants can **define the image's width size**. By default, it is set to 16 pixels since PrestaShop works best with 16x16 pixels image. Images are expected to be resized if they are uploaded bigger. If merchants want to use bigger pictures, this option enables them to set the exact size required. Entering '0' makes PrestaShop use the image's original width size. A tooltip is available: `Image width in pixels. Enter "0" to use the original size.` in `Admin.Shopparameters.Help`.
+
+Only numbers should be typed in the field. If the input contains specific characters, an error notification is displayed: `Width and height must be numeric values.` in `Admin.Shopparameters.Notification`.
+
+Leaving this field empty sets the default value.
+
+**Image height.** Merchants can **define the image's height size**. By default, it is set to 16 pixels since PrestaShop works best with 16x16 pixels image. Images are expected to be resized if they are uploaded bigger. If merchants want to use bigger pictures, this option enables them to set the exact size required. Entering '0' makes PrestaShop use the image's original height size. A tooltip is available: `Image height in pixels. Enter "0" to use the original size.` in `Admin.Shopparameters.Help`.
+
+Only numbers should be typed in the field. If the input contains specific characters, an error notification is displayed: `Width and height must be numeric values.` in `Admin.Shopparameters.Notification`.
+
+Leaving this field empty sets the default value.
+
+
+## Multistore behavior
+
+As the Add/Edit form does not display any shop association block, **when merchants add a social title, it is added to all shops** - no matter the context selected. So all social titles should be listed here, be it an all shops, a group, or a specific shop context.
```

Comment:
```suggestion
As the Add/Edit form does not display any shop association block, **when merchants add a social title, it is added to all shops** - no matter the context selected. 

An information message should be displayed at the top of the page to inform the user of this behavior: "Note that this feature is available in all shops context only. It will be added to all your stores." (_See issue [#19415](https://github.com/PrestaShop/PrestaShop/issues/19415)_

So all social titles should be displayed in the listing, be it an all shops, a group, or a specific shop context.
```